src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+# PostgreSQL database configuration
+
+spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
+spring.datasource.username=${DB_USERNAME}
+spring.datasource.password=${DB_PASSWORD}
+
+# Hibernate properties
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+secret-key=8ffb0df0e3db0551b3b2b9fa3dad501a6582e3a3d7c82a53f4273c786492922c
```

Comment:
I will rather take this secret key out of here and have it in an environment variable

Reply:
alright